modules/programs/autorandr.nix

```diff
@@ -72,6 +72,13 @@ let
         default = "";
         example = "1.0:0.909:0.833";
       };
+
+      rotate = mkOption {
+        type = types.string;
```

Comment:
From the xrandr man page I get the impression that rotate only can have the values 'normal', 'left', 'right', or 'inverted'. If that's the case then it's better to have
```nix
type = types.enum [ "normal" "left" "right" "inverted" ]
```